src/player/PlayerComponent.cpp

```diff
@@ -648,6 +655,49 @@ void PlayerComponent::handleMpvEvent(mpv_event *event)
       }
       break;
     }
+    case MPV_EVENT_HOOK:
+      // FIXME: do these actually need to be filtered on old libmpv versions?
```

Comment:
Do these actually need to be filtered out on older libmpv versions? (IOW, will the old API generate both "message" events and "EVENT_HOOK" events?)

Reply:
The old message events are only generated for legacy hooks.

This block should be preprocessor-`#if`'d as well.